src/main/java/nextstep/subway/line/domain/SectionList.java

```diff
@@ -0,0 +1,164 @@
+package nextstep.subway.line.domain;
+
+import nextstep.subway.station.domain.Station;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+@Embeddable
+public class SectionList {
+    @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST, CascadeType.MERGE}, orphanRemoval = true)
+    private List<Section> sections = new ArrayList<>();
+
+    /**
+     * 생성자
+     */
+    protected SectionList() {
+    }
+
+    public SectionList(List<Section> sections) {
+        this.sections = sections;
+    }
+
+    /**
+     * 비즈니스 메소드
+     */
+    public void addToCreateLine(Section section) {
```

Comment:
```suggestion
    public void add(Section section) {
```

의도를 파악하기 어려운 메서드명입니다.
메서드명에 ToCreateLine를 붙일 필요가 있나요?
클래스는 Section 리스트 용도인데 Line 객체에서만 사용하기 위한 메서드를 뜻하는건가요?
만약 그렇다면 메서드에 Line과 관련된 어떤 제약이 있어야 하는데 아무런 제약도 없습니다.
그냥 클라이언트가 section 객체를 추가하면 끝납니다.
메서드명이 잘못된 의미를 표현하고 있으니 바꿔주는게 좋겠네요

Reply:
메소드명이 노선을 생성하기위한 구간등록이라는 의미로 명명해봤습니다😂
저 메소드가 열려있지만, 사실 노선을 생성하는 용도로만 사용하는 메소드라서 그냥 add()로 했을 때, 다른 개발자들이 볼 때 혼동을 줄 수 있을 것 같아 이름을 변경하였습니다.
노선에 구간을 추가하고 싶다면 addSection을 사용해야합니다.

이름을 다시 add로 변경하기보다는 제약사항으로 sections size가 0일 때만 동작하도록 수정해도 좋지만

지금보니 addToCreateLine과 addSection을 그냥 addSection에서 전부 처리해도 괜찮다는 생각이들어 일단 그렇게 수정해보겠습니다.